include/openthread/link.h

```diff
@@ -1204,6 +1204,29 @@ otError otLinkSetRegion(otInstance *aInstance, uint16_t aRegionCode);
  */
 otError otLinkGetRegion(otInstance *aInstance, uint16_t *aRegionCode);
 
+/**
+ * The Wake-up channel.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ *
+ * @returns The Wake-up channel.
+ *
+ */
+uint8_t otLinkGetWakeupChannel(otInstance *aInstance);
+
+/**
+ * Sets the Wake-up channel.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ * @param[in]  aChannel   The Wake-up sample channel. Channel value should be `0` (Set Wake-up Channel unspecified)
+ *                        or within the range [1, 10] (if 915-MHz supported) and [11, 26] (if 2.4 GHz supported).
+ *
```

Comment:
What's the action if the wake-up channel is set to `0`?  The WC/WED will use the pan channel as the wake-channel, or randomly select a channel as the wake-up channel, or the WC/WED will be disabled or other actions?